lms/static/sass/views/_login-register.scss

```diff
@@ -13,10 +24,24 @@
     $third-party-button-height: ($baseline*1.75);
 
     h2 {
-        @extend %t-title5;
+        @extend %t-title4;
+        margin: 0;
+        letter-spacing: normal;
+        font-family: $sans-serif;
+        color: $uxpl-blue-hover-active;
+    }
+
+    h3 {
+        @extend %t-title6;
+        @extend %t-light;
         margin: 0;
         letter-spacing: normal;
         font-family: $sans-serif;
+        color: #111111;
```

Comment:
Yep, I'm happy to use named colours.  I just wasn't able to find any that matched the design spec exactly, and wasn't sure how strict it was meant to be.

What I found was a [set of tints (lighter) and shades (darker) on `$gray`](https://github.com/edx/edx-platform/blob/master/lms/static/sass/partials/base/_variables.scss#L58-L69):
 
```sass
$gray-l1:   tint($gray,20%)  !default; // #919191 rgb(145,145,145)
$gray-l2:   tint($gray,40%)  !default; // #adadad rgb(173,173,173)
$gray-l3:   tint($gray,60%)  !default; // #c8c8c8 rgb(200,200,200)
                                       // #d9d9d9 rgb(217,217,217)
$gray-l4:   tint($gray,80%)  !default; // #e4e4e4 rgb(228,228,228)
$gray-l5:   tint($gray,90%)  !default; // #f1f1f1 rgb(241,241,241)
$gray-l6:   tint($gray,95%)  !default; // #f8f8f8 rgb(248,248,248)
$gray-l7:   tint($gray,99%)  !default; // #fefefe rgb(254,254,254)
$gray:      rgb(118,118,118) !default; // #767676 rgb(118,118,118)
$gray-d1:   shade($gray,20%) !default; // #5e5e5e rgb(94,94,94)
$gray-d2:   shade($gray,40%) !default; // #474747 rgb(71,71,71)
                                       // #414141 rgb(65,65,65)
$gray-d3:   shade($gray,60%) !default; // #222222 rgb(34,34,34)
$gray-d4:   shade($gray,80%) !default; // #181818 rgb(24,24,24)
                                       // #111111 rgb(17,17,17)
```

Commit 4870d73 uses the closest named colours available to those requested.